config.toml

```diff
@@ -58,3 +58,7 @@ file_name="rust_linked_list.pdf"
 [Books.RustPractiseQuestions]
 print_url="https://rust-unofficial.github.io/rust-practise-questions/print.html"
 file_name="rust_practise_questions.pdf"
+
+[Books.The_Rust_DiscoverySTM]
+print_url="https://docs.rust-embedded.org/discovery/f3discovery/print.html"
+file_name="f3discovery.pdf"
```

Comment:
Could the filename be changed slightly to make it easier to understand at first glance, something like, "microcontroller_rust.pdf"?
I'm not 100% sure about the naming I presented though

Reply:
The name `stm32f3discovery_rust.pdf` might be more informative that's true!!! :100: 

**Although**, because there're two Embedded Rust books using this microcontroller I think that we should also state that this book is for beginners with Embedded, so that people won't confuse it with `The Embedded Rust Book`

so maybe `stm32f3discovery_beginner_rust.pdf`?

The book itself is names '"Discovery"'. 
We could also **add another** version of the 'Discovery' book that uses micro:bit

```
```

Reply:
OH I CLICKED `ENTER` KEY ACCIDENTLY I'M SORRY!!! anyway:

```
[Books.The_Rust_Discovery-micro-bit]
print_url="https://docs.rust-embedded.org/discovery/microbit/print.html"
file_name="micro-bit_beginner_rust.pdf"
```

Reply:
I think the naming proposed by @Cloufish makes more sense than what I initially proposed.